tsk/auto/tsk_auto.h

```diff
@@ -108,7 +108,12 @@ class TskAuto {
     void setFileSystemPassword(std::string fileSystemPassword) { m_fileSystemPassword = fileSystemPassword; }
 
     /**
-     * TskAuto calls this method before it processes the volume system that is found in an
+     * @returns A password that will be used when trying to open each file system or empty.
+     */
+    std::string getFileSystemPassword() const { return m_fileSystemPassword; }
+
```

Comment:
Do we want to change this to a `const std::string &` ?

Reply:
Yes, it should be.